https://github.com/CoreELEC/CoreELEC/pull/208
fix si2168 cmd timeout

Some demodulator si2168 commands may take 130-140 ms
(DVB-T/T2 tuner MyGica T230C v2).

--- a/drivers/media/dvb-frontends/si2168.c
+++ b/drivers/media/dvb-frontends/si2168.c
@@ -76,7 +76,7 @@
 
 	if (cmd->rlen) {
 		/* wait cmd execution terminate */
-		#define TIMEOUT 70
+		#define TIMEOUT 200
 		timeout = jiffies + msecs_to_jiffies(TIMEOUT);
 		while (!time_after(jiffies, timeout)) {
 			ret = si2168_i2c_master_recv_unlocked(client, cmd->args,
